src/System.Private.CoreLib/shared/System/IO/PathInternal.Windows.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Doesn't matter much as it is internal, but since you annotated some, is up to you if you want to annotate this guy as well that can take a null string: https://github.com/dotnet/coreclr/pull/23732/files#diff-deed663847511b7052f321aac368589eR74

Reply:
Fixed